internal/controller/nonadminbackup_controller.go

```diff
@@ -142,5 +148,13 @@ func (r *NonAdminBackupReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 func (r *NonAdminBackupReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&nacv1alpha1.NonAdminBackup{}).
+		Watches(&velerov1api.Backup{}, &VeleroBackupHandler{}).
```

Comment:
how we know the difference between update and patch?

Reply:
For events, everything is called update including patch :)